src/FlightDisplay/FlyViewVideo.qml

```diff
@@ -63,10 +64,21 @@ Item {
         source:         visible ? (QGroundControl.videoManager.uvcEnabled ? "qrc:/qml/FlightDisplayViewUVC.qml" : "qrc:/qml/FlightDisplayViewDummy.qml") : ""
     }
 
+    QGCLabel {
+        text: qsTr("Double-click to exit full screen")
+        font.pointSize: ScreenTools.largeFontPointSize
+        visible: QGroundControl.videoManager.fullScreen && flyViewVideoMouseArea.isHovered
+        anchors.centerIn: parent
+    }
+
     MouseArea {
+        id: flyViewVideoMouseArea
+        property bool isHovered: false
```

Comment:
Why is this property needed? Why can't `flyViewVideoMouseArea.hovered` be used directly?

Reply:
@DonLakeFlyer 
I tried to use the pre-defined `hovered`, but it is not defined for MouseArea. 
([Button](https://doc.qt.io/qt-5/qml-qtquick-controls2-control.html#hovered-prop) has `hovered` property, but it seems [MouseArea](https://doc.qt.io/qt-5/qml-qtquick-mousearea-members.html) does not have it)

Reply:
Also, renaming `isHovered` to `hovered` does not work. (other names than `hovered` works)
I don't figure out the reason, but when `property bool hovered` is defined, `onHoveredChanged` is not triggered.

Reply:
> I tried to use the pre-defined `hovered`, but it is not defined for MouseArea.

That's certainly odd. Looks like it is 'containsMouse` from MouseArea which tells you what you need. You should be able to use that as the reference for showing/hiding the help test.

Reply:
@DonLakeFlyer 
Thank you for the feedback, `containsMouse` worked perfectly. a9c0d52
On the first trial for `containsMouse`, I did not enable `hoverEnabled` which triggers `containsMouse`.

https://user-images.githubusercontent.com/82586164/119072360-4d19eb80-ba26-11eb-92a3-bba7173afee2.mov

Reply:
Great. Much simpler now. Thanks.